liveman_api/lib/liveman_api.ex

```diff
@@ -1,18 +1,44 @@
 defmodule LivemanApi do
   @moduledoc """
-  Documentation for `LivemanApi`.
+  API Wrapper for Liveman
   """
 
-  @doc """
-  Hello world.
+  @timeout 30_000
 
-  ## Examples
+  def get(base_url, url, query \\ %{}) do
```

Comment:
I think we need to header option also to pass the header(for get, post both requests) as for the [API authentication](https://github.com/nimblehq/liveman-demo-api/pull/26) we need to pass `access_token` . What do you think?

Reply:
We will do it later PR